033-OSSDatabaseMigration/Coach/03-offline-migration.md

```diff
@@ -0,0 +1,119 @@
+# Challenge 3: Offline migration
+
+[< Previous Challenge](./02-size-analysis.md) - **[Home](./README.md)** - [Next Challenge >](./04-offline-cutover-validation.md)
+
+## Coach Tips
+
+When creating Azure Postgres DB, create it in GP or MO tier - as Basic tier does not support private link
+
+The attendees will not be able to connect to Azure DB for PostgreSQL/MySQL from within the container. In order to connect, they will need to add the public IP address to the DB firewall. This is the ip address the container is using for egress to connect to Azure DB. One way to find it is to do this:
+```bash
+
+apt update
+apt install curl
+curl ifconfig.me
+
+```
+Please note there are many 3rd party tools similar to SQL wokbench and dbeaver . There is also mydumper and myloader ( https://centminmod.com/mydumper.html ) to use for MySQL
```

Comment:
Same thing here.... hyperlink the mydumper and myloader text to this URL OR add angle brackets to the url for it to be proper markdown, eg: `<http://blahblah.com>`

Reply:
Fixed